server/src/service/game.service.ts

```diff
@@ -0,0 +1,47 @@
+import { DataSource } from 'apollo-datasource'
+import { ApolloError } from 'apollo-server'
+import { ProducedContext } from '@/context'
+import { Game as GameEntity } from '@/entity'
+import { DbError } from '@/util'
+import { Game } from '@/types'
+import { getManager } from 'typeorm'
+
+export class GameService extends DataSource<ProducedContext> {
```

Comment:
This should extend `LiteDataSource` now:
```suggestion
export class GameService extends LiteDataSource {
```
I think I still need to update this in other services.